.github/workflows/build-search.yml

```diff
@@ -33,6 +33,9 @@ jobs:
       - name: Run Auto Generate Settings
         run: yarn autogenerate-settings
 
+      - name: Run Auto Generate System Tables
+        run: yarn autogenerate-system-tables
+
```

Comment:
We unfortunately can't run it on an action because this documentation also needs to be generated as part of the docs check on ClickHouse/ClickHouse repo. That's why it runs as part of the shell script `autogenerate-settings` which is called as part of `yarn build` which runs in the Docker container on core repo.